src/pymovements/datasets/gazebase.py

```diff
@@ -0,0 +1,66 @@
+"""This module provides an interface to the GazeBase dataset."""
+from pymovements.base import Experiment
+from pymovements.datasets.base import PublicDataset
+
+
+class GazeBase(PublicDataset):
+    """GazeBase dataset :cite:p:`GazeBase`.
+
+    Check the respective paper for details.
+    """
```

Comment:
I thought about this quite a bit when I went over the JuDo dataclass. I think adding a real description here is tough -- maybe we should just add it like here and link the paper / osf / wherever the dataset has any details.

Reply:
maybe even more important would be the unit in which the samples are returned

Reply:
I think a brief description of the datasets is necessary to be a useful feature.

From the JuDo1000 implementation I would say this information would be useful:

- number of users (in case of gazebase I would simply add that not all of them are present across all rounds)
- number of sessions (in case of gazebase the distinction between rounds and sessions need to made clear otherwise the user will be very confused)
- sampling rate
- which eyes are recorded
- in which units is the gaze provided
- which stimuli does the participant see. in case of gazebase a very very brief description (one word) of the stimuli would be useful.

Moreover, all this information would be necessary for a feature I want to add in the future, where the user can specify to read in only a specific user, session, round or stimulus.

Reply:
But right now nothing of this is present in the GazeBase docstring, I'll have to add this and then we can see if it's too verbose.

Reply:
I agree we should add a small docstring documentation of the code. I think your list above seems reasonable